git-cliff/src/lib.rs

```diff
@@ -235,15 +239,24 @@ fn process_repository<'a>(
 	}
 
 	// Set the previous release if needed.
-	if args.latest || args.unreleased {
-		let sub = if args.latest || args.tag.is_some() {
-			2
-		} else {
-			1
-		};
-		if let Some((commit_id, version)) =
-			tags.len().checked_sub(sub).and_then(|v| tags.get_index(v))
-		{
+	if !releases.is_empty() &&
+		releases
+			.first()
+			.and_then(|r| r.previous.as_ref())
+			.and_then(|p| p.version.as_ref())
+			.is_none()
+	{
```

Comment:
Can you add a one-line comment here about what is being checked here? It will be helpful when we come back and look at this code some time later 🐻

Reply:
Done 👍